### DISCUSSION TITLE
Email interval reporting

### DISCUSSION THREAD

**Discussion by rsockanc** [jgyates/genmon#1063] [Questions] [answered]
Hi there, is there a way to set up or request a feature to add interval email reporting during an outage? As an example, receive an email once every x user-configured hour(s) while there’s an outage stating how many hours there has been an outage.

**Comment by jgyates** [comment]
the software will currently send an email when an outage begins and ends for controllers that support monitoring the utility voltage. The ending email contains info on how long the outage lasts and the outage page contains the duration of the current and past outages.

Let me think on this feature. I want to weight the benefit vs the code complexity / maintenance cost. The current outage state and duration are also available via MQTT. Can you tell me more about your use case for this feature?

**Reply by rsockanc** [reply]
Understood; this would be more of a "nice to have".  My unit is 5,753 miles away and there is rarely a person onsite but I do have a person that is 2 hours away that can drive to the location if there is a long outage to see what's going on.  City power is not stable, so its not uncommon to have the unit start several times a week and multiple times in a night when the weather is poor. I do receive the outage start and end emails functioning correctly as you have stated. When I receive the first email of an outage, I try to remember to watch out for the "end of outage" email, however as I receive outage notices frequently for just a few hours, it's easy for me to miss checking, especially if I have had back to back outages overnight.  I have had several times in the past year where I did not receive "end of outage" email for over 24 hours and I forgot to check if it came back online. It would help if I had a pesky email every few hours to state " Warning, I'm still in outage mode!"  Thanks for your help and services on this project!

**Reply by jgyates** [reply, marked as answer]
I added this feature so you can update on the about page and then set the value of the "Outage Recurring Notice Interval" setting on the Advanced Settings page (double click the gear icon in the upper right to access the Advanced Settings page). Set the "Outage Recurring Notice Interval" to a positive number of minutes to be notified, so if you want an hour you can set it to 60. Let me know if you have any issues or questions.
